contributors/contribute-to-a-gerrit-project.md

```diff
@@ -61,43 +63,100 @@ You are redirected to Gerrit. Wait a few seconds for the CLA status to update.
 
 ## Corporate Contributor
 
-EasyCLA presents a field where you confirm your association to the company.
+EasyCLA presents a review agreement link where you confirm your association with the company.
 
-1. Select the Corporate CLA that you want to sign.
+1. Select **Corporate CLA**.
 
-   An agreement field appears:
+   ![Gerrit Corporate CLA Agreement](/.gitbook/assets/corporate-cla.png)
+
+2. Click the **Please review the agreement link** and then click the message link that appears:
+
+   ![Gerrit Sign ICLA Link](../.gitbook/assets/cla-gerrit-icla-proceed-to-sign-cla.png)
+   
+3. Sign in to EasyCLA if you are prompted.
+   
+4. Select **Company**.
+
+   To contribute to this project, you must be authorized under a signed Contributor License Agreement. You are contributing on behalf of    your work for a company.
 
-   ![Gerrit Corporate CLA Agreement](../.gitbook/assets/cla-gerrit-ccla-i-agree.png)
 
-2. Enter **I Agree** in the field and click **SUBMIT**.
-3. Read the Confirmation of Association statement and mark the agreement checkbox.
+5. Continue:
+
+   * [If a **Confirmation of Association with** statement appears](contribute-to-a-gerrit-project.md#if-a-confirmation-of-association-with-statement-appears)
+   * [If a **Company has not sigend CCLA** window appears](contribute-to-a-gerrit-project.md#if-a-company-has-not-sigend-CCLA-window-appears)
+   * [If You are not whitelsited](contribute-to-a-gerrit-project.md#if-you-are-not-whitelisted)
+   * [If Company is not in the list](contribute-to-a-gerrit-project.md#if-company-is-not-in-the-list)
+   
+  
+## If a **Confirmation of Association with** statement appears
+
+1. Read the Confirmation of Association statement and select the checkbox.
 
    ![Gerrit Confirmation of Association](../.gitbook/assets/cla-gerrit-confirmation-of-association.png)
 
-4. Click **CONTINUE**.
+2. Click **CONTINUE**.
 
-   A dialog appears and informs you: You are done!
+   A dialog appears and informs yoy: You are done!
+   
+   ![You are done](../.gitbook/assets/cla-github-you-are-done%20%281%29.png)
 
-5. Click **RETURN TO REPO**.
+3. Click **RETURN TO REPO**.
 
    You are redirected to Gerrit. Wait a few seconds for the CLA status to update or refresh the page.
+   
+## If a **Company has not sigend CCLA** window appears
+
+This window apperas if your comapny has not signed a Corporate CLA with Linux Foundation.
+
+To send an email notification to your company's CLA manager to sign Corporate CLA:
+
+1. Select your email address, that you want your comapny manager to whitelist while signing Corporate CLA, from the **Emial to Authorize** drop-down field.
```

Comment:
"Select your email address from the **Email to Authorize** drop-down field. This is the email address that you want your company manager to whitelist while signing the Corporate CLA."

Reply:
changed